### PR TITLE
Add did:github

### PR DESCRIPTION
**Pull request by OR13**
The github method is meant to make working with DIDs very simple at the cost of trusting Github.com for assisting in resolving DID Documents.

Many developers are familar with Github, and its 2 supported public key cryptosystems, GPG and SSH.

Linked Data Signatures are difficult to work with when operating a server or running a local node of some distributed system / blockchain is a requirement.

The objective of GitHub DID is to encourage contribution to the DID Spec and Linked Data Signatures, and allow rapid development of extensions to these without requiring the use of slow, or complicated more trustless infrastructure, such as blockchains or other distributed systems.

GitHub DID is hosted by DIF: https://github.com/decentralized-identity/github-did/

### CODE REVIEW

**Comment by msporny** [comment]
Hey @OR13, sorry the review took so long. While you link to an operational DID Method with a README.md, it doesn't follow the expectations set out by the DID Core spec. Namely, that you clearly define the syntax for the DID Method, the CRUD operations, and a security and privacy considerations section. If you look at many of the other DID Methods in the registry, they define at least those.

At the same time, your implementation is more complete and helpful, but given that we've been attempting to apply the same requirements to everyone across the board (DID syntax, CRUD ops, security and privacy considerations), please update your DID Method specification to cover at least those items. You could just update the README to include those sections.

The PR also now has conflicts given other merges that have happened to the registry.

**Comment by gjgd** [comment]
@msporny same comment: https://github.com/w3c-ccg/did-method-registry/pull/76#issuecomment-578594536

File: index.html
Comment (msporny) [inline index.html]:
```suggestion
          <a href="https://github.com/decentralized-identity/github-did/blob/master/packages/docs/did-method-spec/spec.md">GitHub DID Method</a>
```

**Review by msporny — CHANGES_REQUESTED** [review]
Please link directly to the spec, otherwise, LGTM, merging after you apply the code suggestion.

File: index.html
Comment (msporny) [reply index.html]: Also, please fix merge conflicts if there are any remaining.

**Comment by msporny** [comment]
Merge conflicts appeared after merging did:elem, please fix and we're good to go.

**Comment by gjgd** [comment]
done

**Comment by gjgd** [comment]
done

**Pull request merged**
